ipwb/indexer.py

```diff
@@ -272,7 +271,7 @@ def getCDXJLinesFromFile(warcPath, **encCompOpts):
                     (hstr, payload, nonce) = \
                         encrypt(hstr, payload, encryptionKey)
 
-            # print('Adding {0} to IPFS'.format(entry.get('url')))
+            # print('Adding {entry.get('url')})
```

Comment:
```suggestion
            # print(f'Adding {entry.get("url")} to IPFS')
```

Reply:
Oof, good catch! A second set of eyes is definitely helping here. :-)

Reply:
If it was not a commented out line, you would have caught it in the first place I guess.